src/bindable-property.js

```diff
@@ -248,3 +292,18 @@ export class BindableProperty {
     observer.selfSubscriber = selfSubscriber;
   }
 }
+
+/**
+ * @private
+ * Used for avoid creating mapping function multiple times
+ * @param {Element} element 
+ * @param {string} propertyName 
+ * @param {any} newValue 
+ */
+function propToAttr(element, propertyName, newValue) {
+  if (newValue == null) {
```

Comment:
I like this, but I think the rest of the team is linting for strict `===` always.

Reply:
This will be fixed. Was trying to roll it out so a bit lazy for `||` and `===` 😄